service/sqs/api_test.go

```diff
@@ -1,34 +0,0 @@
-// +build integration
```

Comment:
Why is this test getting removed?

Reply:
This test never actually was used, and [did not compile](https://docs.aws.amazon.com/sdk-for-go/api/service/sqs/#DeleteMessageBatchInput). The SDK's make integration cmd never executed this unit since it was in the service folder. The test looks to exist in order to test the modeled trait, but this is tested via the SDK's protocol tests.